### ISSUE TITLE
Region is yellow, with dark text

### ISSUE THREAD

**Issue by dixius99** [dracula/emacs#106]
Just testing out Dracula in Emacs (macOS, Emacs Plus), and noticed that the selected region is yellow, and the text is a single dark (black) colour. From the [Dracula Syntax Highlighting Specification](https://spec.draculatheme.com/#Selection), and how selected text is handled in the VS Code theme, which is noted as a spec-compliant implementation of the theme, I was expecting the region to be closer to what I see in VS Code, with syntax highlighting for text otherwise staying the same. There's a design reason for this in the Emacs theme?

Emacs:
<img width="897" alt="Screenshot 2024-08-18 at 18 52 38" src="https://github.com/user-attachments/assets/e8195408-c97e-4a01-b445-f559d1ea43bb">

VS Code:
<img width="1162" alt="Screenshot 2024-08-18 at 18 53 29" src="https://github.com/user-attachments/assets/3e403573-a3c3-4da0-9366-4b9fe2ca2624">

**Comment by milouse** [comment]
Historical reason only, and nobody complain until now.

I can try to check the feasibility of changing this highlighting color. I’m just afraid of https://www.xkcd.com/1172/ So probably I’ll keep the "old" behavior behind a variable?

**Comment by dixius99** [comment]
I have to admit I have practically not experience with how Emacs themes work. I made a slight enhancement to the [Catppuccin theme](https://github.com/catppuccin/emacs) a while back, but that was more related to the TUI vs. the actual editor window.

By the way, I see that Catppuccin uses Dracula as a base. And for that theme, it looks like the region is more or less in line with what I figured it would look like in Dracula (a slightly lighter shade of grey, and the syntax highlighting retained).
